provisioning/buffer/kubernetes/build.sh

```diff
@@ -21,6 +21,18 @@ envsubst < templates/worker/config-map.yaml > deploy/worker/config-map.yaml
 envsubst < templates/worker/pdb.yaml        > deploy/worker/pdb.yaml
 envsubst < templates/worker/deployment.yaml > deploy/worker/deployment.yaml
 
+# Remove resources requests/limits to fit all pods to the CI environment
+REMOVE_RESOURCES_LIMITS="${REMOVE_RESOURCES_LIMITS:=false}"
+if [[ "$REMOVE_RESOURCES_LIMITS" == "true" ]]; then
+  echo
+  echo "Removing resources requests/limits ..."
+  echo "--------------------------"
+  # In the regexp is backreference "\1", this ensures that only nested keys that follow are deleted
+  find ./deploy -type f \( -iname \*.yml -o -iname \*.yaml \) \
+  -exec echo "{}" \; \
+  -exec perl -i.original -0777pe "s/(\n *)resources:\s*(\1 +.+)+/\1# <<<REMOVED RESOURCES KEY>>>/g" "{}" \;
+fi
+
```

Comment:
Snazil som sa vypnut kontorlu resources limits najprv v `MiniKube/k8s`.

------

Podarilo sa mi presvedcit `kube-scheduler` aby vypol plugin `NodeResourcesFit`.
Teda, pod sa "odoslal" do `kubelet`.
Ale tam som zistil, ze to `kubelet` kontroluje opat, a neda sa to vypnut.

-----

Potom som skusil "oklamat" `MiniKube`, ze ma viac CPU/mem.
Na to sa interne pouziva `cAdvisor`, ale ani ten som nedokazal externe prinutit, aby vratil iny pocet CPU.

-----

Tak mi zostala posledna moznost, odstranit `resources` z definicii, cez regexp.
Odstrani sa riadok `resources` a vsetky vnorene kluce.

![image](https://user-images.githubusercontent.com/19371734/202991045-6e2169f5-aca9-4568-bd1b-62641e5462e7.png)
![image](https://user-images.githubusercontent.com/19371734/202991101-b5b3e285-6515-42b5-9391-8d16250b4d4b.png)